views/pages/courses/index.ejs

```diff
@@ -254,7 +254,7 @@ completedCourses = [{
                   Date: Not Defined Yet
                 </ul>
                 <ul>
-                  Category/s: <%currentCourses[i].more.category.forEach(category => {%>
+                  Category/s: <%currentCourses[i]?.more?.category.forEach(category => {%>

```

Comment:
@Pirundy @RHornberger20 Maybe this should be Not Defined Yet also?

Reply:
Yep didn't catch that thanks!